### PR TITLE
Project Documentation Stoica Stefan Nicolae

### PR DESCRIPTION
**Pull request by bep1s69**
### Pull Request Overview

This pull request the project documentation for Stoica Stefan Nicolae

### Author

Signed-off-by: Stefan Stoica Nicolae <stoicastef27@gmail.com>

### CODE REVIEW

**Comment by alexandruradovici** [comment]
Please write your name in the pull request description.

File: docs/project/2024/stefan.stoica2706/index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Project Name
+Sensor Control System
+
+**Author**: Stoica Stefan Nicolae \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-bep1s69
```
Comment (alexandruradovici) [inline docs/project/2024/stefan.stoica2706/index.md:5]:
```suggestion
:::info

**Author**: Stoica Stefan Nicolae \
**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-bep1s69

:::
```

File: docs/project/2024/stefan.stoica2706/index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
++-----+-----+
+|           |
+Temperature   Light
+Sensor        Sensor
+|           |
+Raspberry Pi Pico ----------- WiFi Module
+|           |                   |
+Motor       LED             Web Server
+(Fan)     (Variable Intensity)   |
+Control Interface
```
Comment (alexandruradovici) [inline docs/project/2024/stefan.stoica2706/index.md:31]:
````suggestion
```
[Main Components]
|
+-----+-----+
|           |
Temperature   Light
Sensor        Sensor
|           |
Raspberry Pi Pico ----------- WiFi Module
|           |                   |
Motor       LED             Web Server
(Fan)     (Variable Intensity)   |
Control Interface
```
````

File: docs/project/2024/stefan.stoica2706/index.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [55 RON](https://robotescu.net/product/raspberry-pi-pico-w/) |
+| [Raspberry Pi Starter Kit] | Comes with useful parts (breadboard, jumper wires, photodiode etc) | [59 RON](https://robotescu.net/product/raspberry-pi-pico-starter-kit/) |
+| [Plusivo Electronics Starter Kit] | Comes with useful parts too (larger breadboard, motor) | [42 RON](https://www.emag.ro/kit-plusivo-pentru-introducere-in-electronica-x0019ajd65/pd/D1MZPDMBM/?ref=history-shopping_358424322_77141_1) |
+| [DS18B20 Temperature sensor] | Uded to measure ambient temperature | [10 RON](https://www.emag.ro/senzor-temperatura-rezistent-la-apa-ds18b20-cl627/pd/D55RPRBBM/?ref=history-shopping_358801318_38837_1) |
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [esp-rs](https://github.com/esp-rs) | Wi-Fi driver | Used for the Wi-Fi connectivity |
```
Comment (alexandruradovici) [inline docs/project/2024/stefan.stoica2706/index.md:79]: This will not work with the Raspberry Pi Pico.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by ZamfirAndreiPaul — APPROVED** [review]

**Pull request merged**